docker/data-science/notebooks/ch07/2.semantic-search.ipynb

```diff
@@ -50,7 +50,7 @@
     "def start_reviews_search_webserver():\n",
     "    stop_running_webservers() #in case it was already running\n",
     "    get_ipython().system = os.system\n",
-    "    ! cd ../webserver && python start-webserver.py &\n",
+    "    ! cd webserver && python start-webserver.py &\n",
```

Comment:
Note to Trey: run an verify this. I'm sure it's right, just want to double check.